.github/workflows/ci.yml

```diff
@@ -341,15 +341,15 @@ jobs:
           earthly --strict \
             --secret=AWS_ACCESS_KEY_ID=${{ secrets.AWS_USR }} \
             --secret=AWS_SECRET_ACCESS_KEY=${{ secrets.AWS_PSW }} \
-            +ci-push-build-artifacts --AWS_DEFAULT_REGION=us-east-1 --CROSSPLANE_VERSION=${CROSSPLANE_VERSION}
+            +ci-push-build-artifacts --AWS_DEFAULT_REGION=us-east-1 --CROSSPLANE_VERSION=${CROSSPLANE_VERSION} --BUILD_DIR=${GITHUB_REF##*/}
```

Comment:
not quite sure of this syntax after reading the `GITHUB_REF` documentation on https://docs.github.com/en/actions/learn-github-actions/variables, could you help me understand a bit better?

For PRs, this will result in `merge`, for master it would be `master`, and for release branches it would be something like `release-1.17`?  That seems to be the scheme under https://releases.crossplane.io/build at least, so I'm hoping that's what this syntax produces 😅 

Also just now realizing that all the `PR-xyz` dirs under https://releases.crossplane.io/build/ are a legacy format and we don't do that anymore after `PR-1858` or so. TIL.

Reply:
@jbw976 Yeah that matches my understanding of the syntax too.

You can see we were using this before I moved us to Earthly, e.g.:

https://github.com/crossplane/crossplane/blob/e68021de9c2d5344b9da13ba1aed2dd722789786/.github/workflows/ci.yml#L418

My guess is that we actually wanted it to create `PR-xyz` style dirs but in practice it was updating `merge` this whole time instead.

I don't feel strongly about whether we match the `merge` behavior or not. I kinda question whether we really need to build and upload all these artifacts for every single PR at all.